plugins/sense-uploader/pom.xml

```diff
@@ -59,7 +59,6 @@
             <groupId>kg.apc</groupId>
             <artifactId>jmeter-plugins-emulators</artifactId>
             <version>0.2</version>
-            <scope>test</scope>
```

Comment:
Why this change? Emulators are not supposed to be used for anything except testing.